--- origsrc/dbus-glib-0.76/Doxyfile.in	2008-05-14 05:21:15.000000000 -0500
+++ src/dbus-glib-0.76/Doxyfile.in	2008-07-06 23:21:40.625000000 -0500
@@ -49,7 +49,7 @@
 #---------------------------------------------------------------------------
 # configuration options related to the input files
 #---------------------------------------------------------------------------
-INPUT                  = dbus
+INPUT                  = @top_srcdir@/dbus
 FILE_PATTERNS          = *.c *.h
 RECURSIVE              = YES
 #EXCLUDE                = test
